lib/feature/auth/login/ui/login_view.dart

```diff
@@ -106,15 +106,17 @@ class LoginView extends StatelessWidget {
   }
 
   void _onSignUpPressed(BuildContext context) {
-    context.read<AuthRouterDelegate>().setSignupNavState();
+    context.read<AuthRouterDelegate>().setSignupUsernameNavState();
   }
 
   onSelected(BuildContext context, String item) {
-    if(item == 'theme'){
-      final themeNotifier = Provider.of<ThemeChangeNotifier>(context, listen: false);
+    if (item == 'theme') {
+      final themeNotifier =
+          Provider.of<ThemeChangeNotifier>(context, listen: false);
```

Comment:
can we replace this with context.read< ThemeChangeNotifier >()?

Reply:
Yes this works, I changed it, as well as the localizationNotifier